tests/test-extensions/marvel-extension/src/main/java/org/mule/test/marvel/drstrange/DrStrangeOperations.java

```diff
@@ -67,6 +74,93 @@ public String readStream(@Connection MysticConnection connection, @Optional(defa
     }
   }
 
+  public PagingProvider<MysticConnection, Relic> getRelics(StreamingHelper streamingHelper) {
+    return new PagingProvider<MysticConnection, Relic>() {
+
+      private int currentPage = 1;
+      private final int PAGES = 4;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make this final field static too. ([squid:S1170](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1170))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7751&issues=AWm78GvHotIXnmsUmYjP&open=AWm78GvHotIXnmsUmYjP)

[ISSUE_KEY]: AWm78GvHotIXnmsUmYjP